src/controllers/showController.test.ts

```diff
@@ -0,0 +1,132 @@
+import * as faker from 'faker';
+
+import { sequelize, Show } from '../models';
+import * as showController from './showController';
+import { ShowInstance } from '../models/Show';
+
+beforeEach(async () => {
+  await sequelize.sync({ force: true });
+});
+
+function getRandomShow() {
+  return {
+    name: faker.name.findName(),
```

Comment:
Lol this should be changed because it's from sources. You may want to checkout the create-data script. We may want to pull some logic out into reusable functions there too?

Reply:
Lamooo sorry